Replace captain's garment bag with CapDrobe (captain's dresser-wardrobe).

alfalfascout opened the pull request:
<!-- By ticking or leaving ticked the option "Allow edits and access to secrets by maintainers" you give permission for repository maintainers to push changes to your branch without explicitly asking. -->

<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## What Does This PR Do

Makes a dresser-wardrobe to hold all the captain's clothes, as well as a base type for anybody else who wants one. They are lockable.

<!-- Include a small to medium description of what your PR changes. -->
<!-- Document all changes, as not doing this may delay reviews or even discourage maintainers from merging your PR! -->
<!-- If your PR fixes an issue, add "Fixes #1234" somewhere in the PR description. This will automatically close the bug upon PR submission. -->

## Why It's Good For The Game

From what I understand, we kept having to increase the maximum size of the garment bag specifically because of added captain gear.

<!-- Add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Images of changes

<img width="468" height="370" alt="captain's dresser wardrobe" src="https://github.com/user-attachments/assets/67f638bd-d681-42e0-bf69-eac1435f6bb1" />

<!-- If you did not make a map or sprite edit, you may delete this section. You may include a gif or mp4 of your feature if you want. -->

## Testing

Compiled, hosted, locked and unlocked wardrobes, took things out, put things in.

<!-- How did you test the PR, if at all? -->

## Declaration

- [x] I confirm that I either do not require [pre-approval](https://github.com/ParadiseSS13/Paradise/blob/master/docs/CODE_OF_CONDUCT.md#types-of-changes-that-need-approval) for this PR, or I have obtained such approval and have included a screenshot to demonstrate this below.
  <!-- Replace the box with [x] to mark as complete. -->
  <!-- Ensure there are no spaces between the x and the square brackets [] else this will not work properly. -->

## Changelog

:cl:
add: Added CapDrobe (a dresser wardrobe) and placed one in each captain's office.
del: Removed captain's garment bag.
imageadd: Added CapDrobe sprites.
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->
<!-- If a PR has no impact on players (i.e. a code refactor that does not change functionality) then the entire Changelog heading and contents can be removed. -->

Christasmurf commented:
<img width="497" height="368" alt="image" src="https://github.com/user-attachments/assets/b896347a-1b09-4645-858c-4f4b21d417ba" />
Looks like the icon for the broken state has been misnamed, will provide more comments on code shortly

Christasmurf commented on code/game/machinery/vendors/wardrobe_vendors.dm:
```diff
@@ -764,77 +764,47 @@
 	products = list(/obj/item/clothing/under/rank/civilian/chef = 2,
 					/obj/item/clothing/under/misc/waiter = 2,
 					/obj/item/clothing/suit/chef = 2,
-					/obj/item/clothing/suit/chef/bw = 2,
```
All of your chef gear from a previous PR is being removed here, make sure to merge master

Christasmurf commented on code/game/machinery/vendors/wardrobe_vendors.dm:
```diff
@@ -1170,3 +1140,75 @@
+		/obj/item/clothing/head/caphat/parade = 1,
+		/obj/item/clothing/head/caphat/parade/white = 1,
+		/obj/item/clothing/head/beret/captain = 1,
+		/obj/item/clothing/head/beret/captain/white = 1,
```
I'd change all of the hat options above here to be 2 to account for a replacement Captain mid-shift

Christasmurf commented on code/game/machinery/vendors/wardrobe_vendors.dm:
```diff
@@ -1170,3 +1140,75 @@
+		/obj/item/clothing/under/rank/captain/skirt/white = 60,
+		/obj/item/clothing/under/rank/captain/parade = 100,
+		/obj/item/clothing/under/rank/captain/dress = 100,
+		/obj/item/clothing/gloves/color/captain = 40,
```
I'd make these gloves free of charge like the carapace

Christasmurf reviewed (commented):
Some suggestions, also make sure to include images of map changes (either in game or in the mapping software) on your PR

alfalfascout commented:
Is this the consensus, Design Team? Should I make them all free?

Qwertytoforty commented:
At this time I don't think there should be a vendor myself. Garmet bags to me make more sense than vendors for heads (personaly), don't take up more mapping space, and are a little bonus for command with their job options being free

Stoniest commented:
I think Garmet bags should have a huge carry size, so it doesn't have to keep being upped every new addition. Probably 32 slots should be more then enough.
(I'm aware you still have to populate the specific garmet bag, but you'd have to do that for the drobe anyway.)

alfalfascout commented:
Previous description preserved below.

<details><summary>Details</summary>
<p>

<!-- By ticking or leaving ticked the option "Allow edits and access to secrets by maintainers" you give permission for repository maintainers to push changes to your branch without explicitly asking. -->

<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## What Does This PR Do

Makes a wardrobe vendor for the captain so we stop overflowing their garment bag.

I have never played captain and I am not 100% about my mapping changes, so if you have suggestions then PR this branch or leave a comment. This comes out of draft when someone knowledgeable tells me it's ready.

<!-- Include a small to medium description of what your PR changes. -->
<!-- Document all changes, as not doing this may delay reviews or even discourage maintainers from merging your PR! -->
<!-- If your PR fixes an issue, add "Fixes #1234" somewhere in the PR description. This will automatically close the bug upon PR submission. -->

## Why It's Good For The Game

From what I understand, we kept having to increase the maximum size of the garment bag specifically because of added captain gear.

<!-- Add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Images of changes

<img width="609" height="715" alt="2025-11-05 14_12_50-Paradise Station 13" src="https://github.com/user-attachments/assets/abedb9df-18e6-40c8-9610-d632c5bd13b3" />
<img width="456" height="437" alt="2025-11-05 14_13_02-Paradise Station 13" src="https://github.com/user-attachments/assets/159c1484-e84c-4024-ab14-e6bd0ef8b58a" />


<!-- If you did not make a map or sprite edit, you may delete this section. You may include a gif or mp4 of your feature if you want. -->

## Testing

Compiled, hosted, browsed the CapDrobe inventory, hacked the machine, tipped it over, bought things.

<!-- How did you test the PR, if at all? -->

## Declaration

- [x] I confirm that I either do not require [pre-approval](https://github.com/ParadiseSS13/Paradise/blob/master/docs/CODE_OF_CONDUCT.md#types-of-changes-that-need-approval) for this PR, or I have obtained such approval and have included a screenshot to demonstrate this below.
  <!-- Replace the box with [x] to mark as complete. -->
  <!-- Ensure there are no spaces between the x and the square brackets [] else this will not work properly. -->

## Changelog

:cl:
add: Added CapDrobe and placed one in each captain's office.
del: Removed captain's garment bag.
imageadd: Added CapDrobe sprites.
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->
<!-- If a PR has no impact on players (i.e. a code refactor that does not change functionality) then the entire Changelog heading and contents can be removed. -->


</p>
</details>

DGamerL commented on code/game/objects/structures/dresser.dm:
```diff
@@ -7,56 +7,59 @@
 	anchored = TRUE
 
 /obj/structure/dresser/attack_hand(mob/user as mob)
```
```suggestion
/obj/structure/dresser/attack_hand(mob/user)
```

DGamerL commented on code/game/objects/structures/dresser.dm:
```diff
@@ -77,3 +80,133 @@
+		add_fingerprint(H)
+		update_appearance(UPDATE_OVERLAYS)
+	else
+		to_chat(user, "<span class='notice'>Access Denied.</span>")
```
Can you invert the if-statement here?

DGamerL reviewed (commented):
Should be almost ready for merge

alfalfascout commented on code/game/objects/structures/dresser.dm:
```diff
@@ -77,3 +80,133 @@
+/obj/structure/dresser/wardrobe/AltClick(mob/user as mob)
+	toggle_lock(user)
+
+/obj/structure/dresser/wardrobe/attack_hand(mob/user as mob)
```
Why is it there in the first place is what I want to know. I kept it there because the original procs had it so I thought it was probably necessary somehow, but what does it even do?
